applications.json

```diff
@@ -4315,6 +4315,22 @@
                 "javascript"
             ]
         },
+        {
+            "short_description": "Command-line tool to customize the official Spotify client. Supports Windows, MacOS and Linux.",
+            "categories": [
+                "music"
+            ],
+            "repo_url": "https://github.com/spicetify/spicetify-cli",
+            "title": "spicetify-cli",
+            "icon_url": "",
+            "screenshots": [
+                "https://github.com/spicetify/spicetify-cli/blob/master/.github/assets/logo.png"
```

Comment:
@curiousvlxd Please use link to an image - https://raw.githubusercontent.com/spicetify/spicetify-cli/master/.github/assets/logo.png

Reply:
> @curiousvlxd Please use link to an image - https://raw.githubusercontent.com/spicetify/spicetify-cli/master/.github/assets/logo.png

Done! [commit link](https://github.com/serhii-londar/open-source-mac-os-apps/pull/816/commits/9295b40de2506bed9f1f9f724d357491372d7994)